velox/functions/prestosql/tests/TypeOfTest.cpp

```diff
@@ -69,6 +70,19 @@ TEST_F(TypeOfTest, basic) {
   EXPECT_EQ("qdigest(real)", typeOf(QDIGEST(REAL())));
   EXPECT_EQ("qdigest(double)", typeOf(QDIGEST(DOUBLE())));
 
+  auto enumName = "test.enum.mood";
+  auto enumName2 = "someEnumType";
+  std::unordered_map<std::string, int64_t> enumMap = {
+      {"CURIOUS", -2}, {"HAPPY", 0}};
+  LongEnumParameter longEnumParameter(enumName, enumMap);
```

Comment:
longEnumParameter is long and hard to distinguish from longEnumParameter2

consider renaming to something like 'feeling' and 'mood'.